src/nxdoc/nuxeo-server/administration/database-configuration/documentdb.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Amazon DocumentDB
+labels:
+    - database
+    - documentdb
+tree_item_index: 700
+
+---
+`DocumentDB` is compatible with `MongoDB`, therefore, it can be used by `Nuxeo` like a `MongoDB` backend,
+but there are limitations.
+
+{{#> callout type='warning' heading='elastic clusters are not supported'}}
+Only instance clusters are supported due to elastic clusters' limitations.
```

Comment:
As discussed, it should be rewritten to something like: 
"""
Only DocumentDB "instance cluster" has been successfully tested with Nuxeo.
The DocumentDB "elastic cluster" behaves differently than MongoDB regarding index initialization, at the moment it cannot be used by Nuxeo.
"""
if possible with links to better understand what is "instance" and "elastic" clusters.

Reply:
Added in an info callout as it was getting fat (especially since new lines are ignored):
<img width="634" alt="image" src="https://github.com/user-attachments/assets/d61a2aa3-1725-4a0a-800d-8b0d6e32bfd4">

Reply:
<img width="640" alt="image" src="https://github.com/user-attachments/assets/0d6320f6-73a4-443f-9808-579a05d3ad70">